cluster-provision/k8s-multus/scripts/node01.sh

```diff
@@ -26,9 +26,7 @@ kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /tmp/cna/network-addon
 
 # Wait until flannel cluster-network-addons-operator and core dns pods are running.
 # Wait until all the network components are ready.
-kubectl --kubeconfig=/etc/kubernetes/admin.conf wait networkaddonsconfig cluster --for condition=Ready --timeout=800s
-
-kubectl --kubeconfig=/etc/kubernetes/admin.conf create -f /tmp/kubernetes-ovs-cni.yaml
```

Comment:
why we do not need it anymore?

Reply:
because ovs is now part of the cluster-network-addons-operator